_format/1.1/index.md

```diff
@@ -361,9 +381,19 @@ A "resource identifier object" is an object that identifies an individual
 resource.
 
 A "resource identifier object" **MUST** contain `type` and `id` members.
+The only exception is that the `id` member is not required when the resource
+identifier object contains a `lid` that identifies it as matching a new resource
+to be created on the server.
```

Comment:
Similar to the comment on the exception above - can we make every sentence true on its own? Maybe:
> A "resource identifier object" **MUST** contain a `type` member. A resource identifier object must also contain an `id` member, except when it contains a `lid` that identifies it as matching a new resource to be created on the server.

Reply:
Yes, I think that is an improvement.